kpet/__init__.py

```diff
@@ -98,6 +98,18 @@ def build_run_command(cmds_parser, common_parser):
         default=[],
         help='List of mbox URLs/paths comprising the patch series'
     )
+    print_test_cases_parser = action_subparser.add_parser(
+        "print_test_cases",
```

Comment:
Could you maybe name it `print-test-cases`? Dashes are easier to type and aesthetically look more like a command, than a debug hack, like something internal.

Not necessary for my ACK.

Reply:
Done